server/session_ws.go

```diff
@@ -164,7 +166,7 @@ func (s *sessionWS) Consume(processRequest func(logger *zap.Logger, session Sess
 			if !websocket.IsCloseError(err, websocket.CloseNormalClosure, websocket.CloseGoingAway, websocket.CloseNoStatusReceived) {
 				// Ignore underlying connection being shut down while read is waiting for data.
 				if e, ok := err.(*net.OpError); !ok || e.Err.Error() != "use of closed network connection" {
-					s.logger.Debug("Error reading message from client", zap.Error(err))
+					s.logger.Warn("Error reading message from client", zap.Error(err))
```

Comment:
At high concurrent user counts there will always be some number of disconnects expected, usually due to end device network connectivity problems. This is logged at debug level because it's not a problem for the server or something for the developer/operator to fix, and will cause a large amount of log noise.

I suggest we leave this at debug level.

Reply:
OK. I think you are right. If the network of clients are bad, it will log a lot.